--- mod_epp.h	2015-03-27 15:30:20.027130050 +0200
+++ mod_epp_new.h	2015-03-27 15:30:38.499130269 +0200
@@ -60,7 +60,7 @@
 #endif
 
 #define EPP_TCP_HEADER_SIZE 4		/* just one longword */
-#define EPP_MAX_FRAME_SIZE 65536	/* don't accept larger xml data blocks */
+#define EPP_MAX_FRAME_SIZE 15728640 /* don't accept larger xml data blocks (8MB + 30% base64 + xml + savings)*/
 #define TRIDSIZE 128			/* actually, it should be 3 to 64 chars,
 					   but due to unicode we'll give it more room. */
 
